pkg/parallel/parallelctr/ctr.go

```diff
@@ -0,0 +1,42 @@
+package parallelctr
```

Comment:
Why stutter name on enclosing directory and file?

Reply:
Because I am not very creative and could not think of a better name.